mmv1/products/cloudrunv2/Job.yaml

```diff
@@ -437,6 +437,126 @@ properties:
                   type: String
                   description: |-
                     Container's working directory. If not specified, the container runtime's default will be used, which might be configured in the container image.
+                - name: 'dependsOn'
+                  type: Array
+                  description: |-
+                    Names of the containers that must start before this container.
+                  item_type:
+                    type: String
+                - name: 'startupProbe'
+                  type: NestedObject
+                  description: |-
+                    Startup probe of application within the container.
+                    All other probes are disabled if a startup probe is provided, until it
+                    succeeds. Container will not be added to service endpoints if the probe fails.
+                  default_from_api: true
+                  properties:
+                    - name: 'initialDelaySeconds'
+                      type: Integer
+                      description: |-
+                        Number of seconds after the container has started before the probe is
+                        initiated.
+                        Defaults to 0 seconds. Minimum value is 0. Maximum value is 240.
+                      default_value: 0
+                    - name: 'timeoutSeconds'
+                      type: Integer
+                      description: |-
+                        Number of seconds after which the probe times out.
+                        Defaults to 1 second. Minimum value is 1. Maximum value is 3600.
+                        Must be smaller than periodSeconds.
+                      default_value: 1
+                    - name: 'periodSeconds'
+                      type: Integer
+                      description: |-
+                        How often (in seconds) to perform the probe.
+                        Default to 10 seconds. Minimum value is 1. Maximum value is 240.
+                      default_value: 10
+                    - name: 'failureThreshold'
+                      type: Integer
+                      description: |-
+                        Minimum consecutive failures for the probe to be considered failed after
+                        having succeeded. Defaults to 3. Minimum value is 1.
+                      default_value: 3
+                    - name: 'tcpSocket'
+                      type: NestedObject
+                      description: |-
+                        TcpSocket specifies an action involving a TCP port.
+                      send_empty_value: true
+                      allow_empty_object: true
+                      exactly_one_of:
+                        - 'template.0.containers.0.startup_probe.0.tcp_socket'
```

Comment:
I think these should technically be `template.0.template.0.containers.0...`. I don't think it works anyway because containers is a list (so we can't know which index to look at), but I did notice this was on the other cloudrun fields and I suppose serves to document the intention. The other option would be to simply remove them.